### ISSUE TITLE
Unexpected behaviour

### ISSUE THREAD

**Issue by camaramm** [GiovineItalia/Gadfly.jl#643]
I'm simply trying Gadfly with the following commands:

``` julia
   > import Gadfly
   > Gadfly.plot(x=rand(10), y=rand(10))
```

But julia session crashes and keeps repeating the error message:

```
   >** (pcmanfm:#####): WARNING **: The directory '~/Templates' doesn't exist, ignoring it
   >fixme:exec::SHELL_execute flags ignored: 0x00000100
```

where `#####` stands for some integer.
Even after quitting Julia, my terminal session keeps repeating the error message, which just stops after killing process `wineserver'.
However, Gadfly works fine when running under Juno, in spite of being very slow.
I'm running Julia Version 0.3.10 on x86_64-linux-gnu (Arch Linux).

**Comment by vchuravy** [comment]
Hej,

could you try: 

````julia
import Gadfly
p = Gadfly.plot(x=rand(10), y=rand(10))
display(p)
````

And what does `Gadfly.default_mime()` give you?

I suspect that the problem lies in the fact that pcmanfm is trying to open a html site.

You can use the `extra/perl-file-mimeinfo` package to check and set the default program for html pages.

````
touch test.html
mimeopen -d test.html
````

**Comment by camaramc** [comment]
Hello vchuravy,

Your suggestion involving  `display(p)` didn't work as well.
But your suspicion about opening html was right: `Gadfly.default_mime()` returns "text/html".
I've installed extra/perl-fime-mimeinfo and set firefox as default program and it worked, I was able to see the graph.
By the way, is there a program/driver better than firefox to display it?

However, there is still an error message:
```
   ** (pcmanfm:1143): WARNING **: The directory '~/Templates' doesn't exist, ignoring it

   (process:####): GLib-CRITICAL **: g_slice_set_config: assertion 'sys_page_size == 0' failed
```
Is it worth worrying about it?

Thank you for now.

Regards.
